db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -110,16 +110,12 @@ def requested_number_of_cpus(args_hash)
   request_hash_value(args_hash)
 end
 
-def bytes_to_megabytes(bytes)
-  bytes / 1024**2
-end
-
 def vmdb_object(model, id)
   $evm.vmdb(model, id.to_i) if model && id
 end
 
 def requested_storage(args_hash)
-  vm_size = bytes_to_megabytes(args_hash[:resource].vm_template.provisioned_storage)
+  vm_size = args_hash[:resource].vm_template.provisioned_storage
```

Comment:
@billfitzgerald0120 
Is the provisioned_storage going to be in bytes for all platforms.